src/components/AudioPlayer/AudioPlayer.component.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { Fragment } from 'react';
+import { useSelector } from 'react-redux';
+
+import PlayerContainer from './Player';
+import AudioComponent from './Audio';
+
+import './style/player.scss';
+import './style/progress.scss';
+
+const AudioPlayerComponent = () => {
+  const darkThemeActive = useSelector((state) => state.settings.darkTheme);
+
+  return (
+    <div className={`audio-container ${darkThemeActive ? 'dark-theme-background' : ''}`}>
+      <AudioComponent />
+
+      <>
```

Comment:
`<Fragment>`. You imported it, but not used it. Did you force the commit? this should have been detected by the linter 🤔